spec/PhpSpec/Util/MethodAnalyserSpec.php

```diff
@@ -35,10 +35,20 @@ function it_identifies_internal_classes_as_non_empty()
     {
         $this->methodIsEmpty('DateTimeZone', 'getOffset')->shouldReturn(false);
     }
+
+    function it_identifies_methods_from_traits()
+    {
+        $this->methodIsEmpty('spec\PhpSpec\Util\ExampleObject', 'emptyMethodInTrait')->shouldReturn(true);
+        $this->methodIsEmpty('spec\PhpSpec\Util\ExampleObject', 'nonEmptyMethodInTrait')->shouldReturn(false);
+    }
 }
 
+require 'ExampleTrait.php';
+
 class ExampleObject
 {
+    use ExampleTrait;
```

Comment:
please use a different class defined in its own file, so that PHP 5.3 can still run all other examples